allvars.h

```diff
@@ -520,7 +520,7 @@ extern struct particle_data
     pthread_spinlock_t SpinLock;
 #endif
 
-    float GravCost;     /*!< weight factor used for balancing the work-load */
+    int GravCost;     /*!< weight factor used for balancing the work-load */
```

Comment:
I thought of doing this, but the hydro cost factor is explicitly a double at the moment:

density.c:238:    P[place].GravCost += All.HydroCostFactor * All.cf.a * remote->Ninteractions;

I don't know why All.cf.a is there - I would be open to just removing it (and thus make GravCost an int), but I haven't had a chance to test what happens if I do.